logic/poi_locations.yaml

```diff
@@ -420,7 +420,7 @@
       top: 0
   logic: checkerboardCave
 - name: Hammers
-  hoverText: Hammers
+  hoverText: <img src='/static/image/hammer.png' class='mini'><img src='/static/image/hammer.png' class='mini'><img src='/static/image/hammer.png' class='mini'><img src='/static/image/hammer.png' class='mini'><img src='/static/image/hammer.png' class='mini'><img src='/static/image/hammer.png' class='mini'><img src='/static/image/hammer.png' class='mini'><img src='/static/image/hammer.png' class='mini'>!!!!!!!!
```

Comment:
so many hammers

Reply:
One for every peg you have to smack with the hammer in that location. 😄